cpp/ps0002.cpp

```diff
@@ -4,35 +4,17 @@ using namespace std;
 
 int main()
 {
-    short Population = 80;
-    short Alternatives = 4;
-    short Teammembers = 10;
-    float Theprice = 5.5;
-    char Thefirstletter = 'N';
-    int Thenumberofbirths = 9557867;
-    double Division = 1.7857142857142;
-    double Electricitybill = 10.99;
-    int unsigned Theage = 3973536000;
-
-
-    cout << "- The population of Amara  : " << Population << endl;
-
-    cout << "- Number of alternatives available at the pharmacy : " << Alternatives << endl;
-
-    cout << "- Team members at a programming company : " << Teammembers << endl;
-
-    cout << "- The price of one oven baked item : " << Theprice << endl;
-
-    cout << "- The first letter of a persons name : " << Thefirstletter << endl;
-
-    cout << "-The number of births in a country with a large popuiation like china : " << Thenumberofbirths << endl;
-
-    cout << "- A more accrate result for dividing 12.5/7 : " << Division << endl;
-
-    cout << "- Electricity bill for a production line in egypt : " << Electricitybill << endl;
-
-    cout << "- The Age of old and young people in seconds : " << Theage << endl;
+    short Population = 80; // 0.75/0.75
+    short Alternatives = 4; // 0.75/0.75
+    short Teammembers = 10; // 0.75/0.75
+    float Theprice = 5.5; // 0.75/0.75
+    char Thefirstletter = 'N'; // 0.75/0.75
+    int Thenumberofbirths = 9557867; // 0.75/0.75
+    double Division = 1.7857142857142; // 0.75/0.75
+    double Electricitybill = 10.99; // 0.75/0.75
+    int unsigned Theage = 3973536000; // 0.75/0.75
+    // total 6.75/6.75
 
     return 0;
-
 }
+// great work 😎 9/10 ⭐

```

Comment:
- [ ] program is in the correct path
- [x] no syntax errors
- [x] formatted code to be clear and readable
- [ ] commented some phrases for blocks of code
+1 bonus mark for your user-friendly program outputting values ⭐